frontend/src/components/FormReview/index.tsx

```diff
@@ -49,32 +63,180 @@ const mockFormData = [
   }
 ];
 
-const FormReview = () => (
-  <div className="form-review">
-    <div className="form-review-title-section">
-      <p className="form-review-title">
-        Form review
-      </p>
-      <Subtitle text="Review data filled in the form (view-only)" />
-    </div>
-    <div>
-      {
+const defaultCode = '16';
+const defaultAgency = '0032 - Strong Seeds Orchard - SSO';
+const defaultPayment = 'ITC - Invoice to client address';
+const agencyOptions = [
+  '0032 - Strong Seeds Orchard - SSO',
+  '0035 - Weak Seeds Orchard - WSO',
+  '0038 - Okay Seeds Orchard - OSO',
+  '0041 - Great Seeds Orchard - GSO',
+  '0043 - Bad Seeds Orchard - BSO'
+];
+
+const orchardMock = {
+  orchardNumber: 123,
+  orchardName: 'Strong seeds orchard',
+  seedlotSpecies: 'SX - Spruce hybrid',
+  femaleGametic: 'F1 - Visual estimate',
+  maleGametic: 'M2',
+  controledCrosses: true,
+  biotechnologicalProcesses: true,
+  polenContamination: true
+};
+
+const interimStorageMock = {
+  applicantAgency: true,
+  interimAgencyName: 'Strong Seeds Orchard - SSO',
+  locationCode: 32,
+  storageStartDate: '2023/01/04',
+  storageEndDate: '2023/01/26',
+  storageLocation: 'Strong Seeds Seed Orchard Company',
+  storageFacilityType: 'VRM'
+};
+
+const ownershipMock = {
+  applicantAgency: true,
+  agencyName: 'Strong seeds orchard',
+  locationCode: 32,
+  ownerPortion: 100,
+  reserved: 100,
+  surplus: 0,
+  fundingSource: 'LFP - Licensee Funded Program',
+  paymentMethod: 'ITC - Invoice to client address'
+};
+
+const FormReview = () => {
+  const [allStepData, setAllStepData] = useState<AllStepData>({
+    interimStep: initInterimState(defaultAgency, defaultCode),
+    ownershipStep: [initOwnershipState(defaultAgency, defaultCode, defaultPayment)],
+    orchardStep: initOrchardState()
+  });
+
+  return (
+    <div className="form-review">
+      <div className="form-review-title-section">
+        <p className="form-review-title">
+          Form review
```

Comment:
I think these string values can already be passed to a `constants.ts` file, look at the Ownership Step for reference. This applies to the string values on the accordions also.